code/game/machinery/computer/abnormality_archive.dm

```diff
@@ -0,0 +1,89 @@
+/obj/machinery/computer/abnormality_archive
+	name = "abnormality archive"
+	desc = "A secure terminal connected to the Lobotomy Corp archive."
+	icon = 'icons/obj/computer.dmi'
+	icon_state = "computer"
+	var/printfile
+	var/printing_status
+	var/notehtml
+	var/note
+	var/list/note_list
+	var/linecount
+	var/paper = 1 //You get one paper
+	var/papermax = 10 //Recycle more paper
+
+/obj/machinery/computer/abnormality_archive/ui_interact(mob/user, datum/tgui/ui)
+	. = ..()
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(!ui)
+		ui = new(user, src, "LC13AbnormalityArchive", name) //apparently connected to .Js which is invisable to dreammaker but not to Visual Studio Code.
+		ui.open()
+
+/obj/machinery/computer/abnormality_archive/ui_static_data(mob/user) //list is created the moment a person boots up the UI and inherited by future iterations of this object
+	. = ..()
+	var/abnormality_info
+	if(!abnormality_info)
+		abnormality_info = list()
+		for(var/_record_detail in subtypesof(/obj/item/paper/fluff/info)) //consists of all threat levels and their branches
+			var/obj/item/paper/fluff/info/record_detail = _record_detail
+			var/list/record_data = list()
+			if(initial(record_detail.name) == "paper" || initial(record_detail.noArchive) == TRUE) //removes template papers and allows for secret files
```

Comment:
Comment may be a bit misleading `... allows for` as an if that filters said property out

Reply:
What? I honestly cant understand what you mean and the subtypes of info allows for all subtypes of info documents. Which currently are threat levels.